### PR TITLE
Refactor supervisor portal UI

### PR DESCRIPTION
**Pull request by bledsoef**
## Issue Description
Fixes issue #440 
  -Making mild changes to the ui of the supervisor portal

## Changes
- Moved the search and clear buttons into the for accordion 
- Changed the location of the download button to the top right and renamed it to "Download Results"
- Added messages to the form types to give a better explanation 
- Added B-numbers to the student column of the table

## Testing
- These were all UI changes. Run flask.

### CODE REVIEW

**Comment by stevensonmichel** [comment]
This is not a bug rather an enhancement. Can you ensure that the Add User button is disabled if neither supervisor nor department is selected after clicking on Add User to Dept?
Screenshot 2024-09-20 155106

File: app/templates/main/supervisorPortal.html [hunk 1]
```diff
@@ -157,54 +160,51 @@ <h1 class="fs-1" data-toggle="collapse">Form Search</h1>
                 <div class="form-check">
                   <input class="form-check-input" name="formType" type="checkbox" value="Labor Status Form" id="original">
                   <label class="form-check-label" for="original">
-                    Original  
+                    <span class="tooltip-right"
+                      data-tooltip="A labor contract">Original
```
Comment (stevensonmichel) [inline app/templates/main/supervisorPortal.html]: This tabulation is not necessary. That same inconsistency is repeated in lines 176, 188, and 200.

**Comment by stevensonmichel** [comment]
Also, after using adding a user, the Add User button gets disabled the next time you opened the modal even though nothing is selected.

File: app/templates/main/supervisorPortal.html (hunk 1 above)
Comment (WackyWeaver) [reply app/templates/main/supervisorPortal.html]: I removed the tabulation!

**Comment by WackyWeaver** [comment]
> Also, after using adding a user, the Add User button gets disabled the next time you opened the modal even though nothing is selected.

I defaulted the button to disabled when the modal button is clicked!

**Comment by WackyWeaver** [comment]
image
The first space is unnecessary and allows people to select "Department" instead of a specific one!

**Comment by stevensonmichel** [comment]
One more edge case: If I select empty department, the button is still activated, and I am able to add that user to the empty department. 
Screenshot 2024-09-27 101745

**Review by hoerstl — APPROVED** [review]
Everything looks peachy

**Pull request merged**